src/test/java/com/github/dockerjava/core/util/CertificateUtilsTest.java

```diff
@@ -0,0 +1,28 @@
+package com.github.dockerjava.core.util;
+
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
+
+public class CertificateUtilsTest {
```

Comment:
We are working on enhancing test coverage. If bug appeared, then it should be covered with test. Not all places are possible to cover atm. This case is pretty easy to cover it, i can handle myself if you want :)